crates/bevy_ecs/src/schedule/executor/mod.rs

```diff
@@ -50,14 +50,16 @@ pub enum ExecutorKind {
 /// [`FixedBitSet`] is used as a smaller, more efficient substitute of `HashSet<usize>`.
 #[derive(Default)]
 pub struct SystemSchedule {
+    // Maps by system id (except `system_ids` field).
```

Comment:
I'm not a fan of grouping things by a single header comment. It's too easy for someone to add a field in the middle of this grouping in the future and then the header comment to be cropped out in the diff for the pr.

We should either have a comment on each field or not bother.